keps/sig-cloud-provider/20191004-out-of-tree-credential-providers.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Out-of-Tree Credential Providers
+authors:
+  - "@mcrute"
+owning-sig: sig-cloud-provider
+participating-sigs:
```

Comment:
I believe this will involve node changes and as such should include sig-node. It may also include security changes and so we should consider adding sig-auth.